content/docs/studio/selfhosted/installation/index.md

```diff
@@ -0,0 +1,7 @@
+## Installing Studio Self-hosted
+
+Enterprise users have the option of hosting Iterative Studio on their premises
+using the following installation methods:
+
+- [AMI (AWS)](/doc/studio/selfhosted/installation/ami)
+- [Helm (Kubernetes)](/doc/studio/selfhosted/installation/helm)
```

Comment:
Should we address here pre-requisites like setting up SCM repo apps before installation?

Reply:
We provide example configurations in each installation guide and suggest looking at the configuration section before deploying so I think it naturally ends up in that order.

For example, the reader might end up on a path like:
`Helm guide -> Configuration -> SCM providers -> GitHub -> Helm guide`

Reply:
I find that the order of operations and an understanding of the high-level steps is something you always want to make crystal clear beforehand and not be "emergent" - it's easy to feel overwhelmed as a reader of a guide if you're jumping back and forth between pages without a high-level ordered-step-list in mind

Reply:
I agree. However, there are going to be problems with circular dependencies either way.

For instance, if we implement your suggestion:
1. User reads the installation methods page and is made aware that Git forges need to be set up before continuing.
2. User heads over to the GitHub page, sets up the GitHub app, and doesn't know what to do when encountering the YAML config section, e.g., https://dvc-org-studio-selfhost-l2oxcs.herokuapp.com/doc/studio/self-hosting/configuration/github#configuring-studio-with-the-github-app

Can we work around this without it becoming confusing for the user? The only idea I have is a separation of the external (e.g., Github) and the internal (Studio) configuration, but this also leads to indirection.